client/src/components/ListCard.js

```diff
@@ -42,12 +43,17 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const ListCard = (props) => {
+  const {list, demoListsArray} = props;
   const { listTitle, itemCount, img } = props.list;
```

Comment:
Can we combine these two lines?

Reply:
I'm not sure, I tried a few different things and it seemed like I couldn't get it to work unless I used two lines like this. I'd definitely prefer it to be one line if possible!

Reply:
I think it'd be `const { { listTitle, itemCount, img }, demoListsArray} = props`, or maybe `const { { ...listTitle, itemCount, img }, demoListsArray} = props`.... but if we can't get it to work, definitely not a problem to do it in two lines

Reply:
I tried both of those suggestions, and neither worked, so I'll keep it as is for now.